### PR TITLE
Draft of tools article

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description

An article about tools.

### CODE REVIEW

**Review by jgerigmeyer — APPROVED** [review]

**Review by jerivas — APPROVED** [review]
Great article! I like the idea of thinking if a tool sits on top of a web technology or completely replaces it

File: content/blog/2022/platform-tools.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+
+I think there's also a tooling component here.
+Many graphic design tools are a step removed
+from the web platform.
+While some do try to export HTML/CSS,
+only a few consider that
+the core goal of the tool.
+Even then, there's been a long-term disconnect
+between a web that's responsive,
+and our almost completely-static design tools.
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:134]:
```suggestion
and our almost completely static design tools.
```

File: content/blog/2022/platform-tools.md [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+      span: full
+tags:
+  - Article
+  - CSS
+  - Sass
+  - Color
+  - Fonts
+  - Design
+summary: |
+  Ask not-just: _How well does it work_?
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:46]:
```suggestion
  Ask not just: _How well does it work_?
```

File: content/blog/2022/platform-tools.md [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+
+Since all legacy colors on the web
+rely on a single color model (RGB)
+and gamut (sRGB),
+there has been little _meaningful_ difference
+between formats (until now).
+Once we used a graphic interface
+to pick the color we want,
+it maybe didn't matter
+what format we copy/paste into
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:199]:
```suggestion
what format we copied/pasted into
```

File: content/blog/2022/platform-tools.md [hunk 4]
```diff
@@ -0,0 +1,497 @@
...
+in any other CSS format.
+But at the end of the day,
+_CSS output is always generated
+in whatever format has the widest browser support_.
+
+It doesn't matter to Sass
+what syntax authors use
+for their _input value_,
+the _output value_ is always
+either HEX or a color-name when opaque,
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:227]:
```suggestion
either HEX or a color name when opaque,
```
possibly an example of a color name in parentheses if what you mean is something like `rebecaapurple`? Not a biggie.

File: content/blog/2022/platform-tools.md [hunk 5]
```diff
@@ -0,0 +1,497 @@
...
+All we can do is wait_.
+
+I even ran into this issue
+by simply hosting my personal site
+on Netlify,
+which provides minification
+for deployed assets like CSS.
+That seems like a great feature!
+But the specific CSS minifier that they use
+is somewhat over-eager,
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:328]:
```suggestion
is somewhat overeager,
```

File: content/blog/2022/platform-tools.md [hunk 6]
```diff
@@ -0,0 +1,497 @@
...
+  alt: |
+    Nested semicircles,
+    from innermost and slowest
+    to outermost and fastest-moving:
+    nature, culture,
+    governance, infrastructure,
+    commerce, fashion
+adactio:
+  caption: |
+    Described by [Jeremy Kieth](https://adactio.com/)
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:18]:
```suggestion
    Described by [Jeremy Keith](https://adactio.com/)
```

File: content/blog/2022/platform-tools.md [hunk 7]
```diff
@@ -0,0 +1,497 @@
...
+  - Sass
+  - Color
+  - Fonts
+  - Design
+summary: |
+  Ask not-just: _How well does it work_?
+  But also: _How well does it fail_?
+  What happens when something goes wrong?
+
+  ---Jeremy Kieth
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:50]:
```suggestion
  ---Jeremy Keith
```

File: content/blog/2022/platform-tools.md [hunk 8]
```diff
@@ -0,0 +1,497 @@
...
+to try and be _clever_ about minification.
+I don't know that a CSS minifier
+should do much more than white-space management.
+
+## Other tools that might intervene
+
+In regard to web fonts,
+Bram points out
+that authors are not taking full advantage
+of variable fonts --
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:354]:
```suggestion
of variable fonts ---
```

File: content/blog/2022/platform-tools.md [hunk 9]
```diff
@@ -0,0 +1,497 @@
...
+
+Most of us are also using
+the over-complicated `font-feature-settings` property,
+instead of the simpler `font-variant`.
+I've been doing that on my site as well!
+
+Why did I think that was the best approach?
+Likely because that's the generated output
+recommended by
+[Wakamai Fondue](https://wakamaifondue.com/) --
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:379]:
```suggestion
[Wakamai Fondue](https://wakamaifondue.com/) ---
```

File: content/blog/2022/platform-tools.md [hunk 10]
```diff
@@ -0,0 +1,497 @@
...
+But that can only happen
+if the tools are designed to
+sit on top of their underlying languages.
+Once the tools stand _between_ us and the language,
+we become entirely reliant on tool-builders
+to determine what features are available.
+
+When our primary tools
+intervene so strongly
+between us and our 'materials' --
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:443]:
```suggestion
between us and our 'materials' ---
```

File: content/blog/2022/platform-tools.md [hunk 11]
```diff
@@ -0,0 +1,497 @@
...
+the core web languages --
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:444]:
```suggestion
the core web languages ---
```

File: content/blog/2022/platform-tools.md [hunk 12]
```diff
@@ -0,0 +1,497 @@
...
+what tools are useful for other web developers.
+But it is my job to
+help people think about the CSS choices they make.
+So in that spirit, please consider:
+
+⚠️🚨⚠️
+When tools intervene
+between you and your access to the web platform,
+**proceed with caution**.
+Ask not-only: _How well does it work?_
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:493]:
```suggestion
Ask not only: _How well does it work?_
```

File: content/blog/2022/platform-tools.md [hunk 13]
```diff
@@ -0,0 +1,497 @@
...
+But also: _How well does it fail?_
+Not-only: _What features do they provide?_
```
Comment (dvdherron) [inline content/blog/2022/platform-tools.md:495]:
```suggestion
Not only: _What features do they provide?_
```

**Review by dvdherron — APPROVED** [review]
Some copy edits. Great read!

File: content/blog/2022/platform-tools.md (hunk 8 above)
Comment (jgerigmeyer) [reply content/blog/2022/platform-tools.md:354]: Why here and not elsewhere?

File: content/blog/2022/platform-tools.md (hunk 11 above)
Comment (jgerigmeyer) [reply content/blog/2022/platform-tools.md:444]: I'm curious about these -- I don't think we use `---` elsewhere on the site for anything other than quote attributions?

Comment (dvdherron) [reply content/blog/2022/platform-tools.md:444]: Ah, you're right. The styleguide specifically forbirds em dashes for asides. I'll remove these suggestions!

File: content/blog/2022/platform-tools.md [hunk 14]
```diff
@@ -0,0 +1,497 @@
...
+I don't think that's the only factor.
+It's possible authors have settled on flexbox as _good-enough_,
+and they don't think it's worth their time
+learning another technique.
+I can understand that,
+but it's unfortunate.
+I think people are missing out
+on one of the coolest features
+added to CSS _ever_.
+Oh well.
```
Comment (stacyk) [inline content/blog/2022/platform-tools.md:410]:
```suggestion

```

I don't think you need the "Oh well" here. Don't give up on these people who haven't seen the light! They will come around.

**Review by stacyk — APPROVED** [review]
Love the entire article.

File: content/blog/2022/platform-tools.md [hunk 15]
```diff
@@ -0,0 +1,497 @@
+---
+draft: true
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/blog/2022/platform-tools.md:2]: copy-pasta from my site.

File: content/blog/2022/platform-tools.md [hunk 16]
```diff
@@ -0,0 +1,496 @@
...
+
+Since all legacy colors on the web
+rely on a single color model (RGB)
+and gamut (sRGB),
+there has been little _meaningful_ difference
+between formats (until now).
+Once we used a graphic interface
+to pick the color we want,
+it maybe didn't matter
+what format we copied/pasted into
```
Comment (jgerigmeyer) [inline content/blog/2022/platform-tools.md:199]: I think the past participle form is `copy-pasted`?

```suggestion
what format we copy-pasted into
```

File: content/blog/2022/platform-tools.md [hunk 17]
```diff
@@ -0,0 +1,495 @@
+---
+title: When Our Tools Hold Us Back
+sub: What happens if the 'pace layers' get out of sync?
+date: 2022-11-10
```
Comment (jgerigmeyer) [inline content/blog/2022/platform-tools.md:4]:
```suggestion
date: 2022-11-11
```

**Review by jgerigmeyer — APPROVED** [review]
@mirisuzanne Looks good! Up to you whether you want to merge now, or update the date and merge Monday.

**Comment by mirisuzanne** [comment]
I'm gonna figure it's fine to go live, even if we want to do more sharing on monday

**Pull request merged**